app/client/cli/actor.go

```diff
@@ -0,0 +1,318 @@
+		Args:  cobra.ExactArgs(4),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO(#150): update when we have keybase
+			pk, err := readEd25519PrivateKeyFromFile(privateKeyFilePath)
+			if err != nil {
+				return err
+			}
+
+			amount := args[1]
+			err = validateStakeAmount(amount)
+			if err != nil {
+				return err
+			}
+			// removing all invalid characters from rawChains argument
+			rawChains := rawChainCleanupRegex.ReplaceAllString(args[2], "")
+			chains := strings.Split(rawChains, ",")
+			serviceURI := args[3]
+
+			// TODO (team): passphrase is currently not used since there's no keybase yet, the prompt is here to mimick the real world UX
+			pwd = readPassphrase(pwd)
+
+			msg := &typesUtil.MessageEditStake{
+				Address:    pk.Address(),
+				Chains:     chains,
+				Amount:     amount,
+				ServiceUrl: serviceURI,
+				Signer:     pk.Address(),
+				ActorType:  cmdDef.ActorType,
+			}
+
+			tx, err := prepareTxJson(msg, pk)
+			if err != nil {
+				return err
+			}
+
+			resp, err := postRawTx(cmd.Context(), pk, tx)
+			if err != nil {
+				return err
+			}
+			// DISCUSS(#310): define UX for return values - should we return the raw response or a parsed/human readable response? For now, I am simply printing to stdout
+			fmt.Println(resp)
+
+			return nil
+		},
+	}
+	return editStakeCmd
+}
+
+func newUnstakeCmd(cmdDef actorCmdDef) *cobra.Command {
+	unstakeCmd := &cobra.Command{
+		Use:   "Unstake <fromAddr>",
+		Short: "Unstake <fromAddr>",
+		Long:  fmt.Sprintf(`Unstakes the prevously staked tokens for the %s actor with address <fromAddr>`, cmdDef.Name),
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO(#150): update when we have keybase
+			pk, err := readEd25519PrivateKeyFromFile(privateKeyFilePath)
+			if err != nil {
+				return err
+			}
+
+			// TODO (team): passphrase is currently not used since there's no keybase yet, the prompt is here to mimick the real world UX
+			pwd = readPassphrase(pwd)
+
+			msg := &typesUtil.MessageUnstake{
+				Address:   pk.Address(),
+				Signer:    pk.Address(),
+				ActorType: cmdDef.ActorType,
+			}
+
+			tx, err := prepareTxJson(msg, pk)
+			if err != nil {
+				return err
+			}
+
+			resp, err := postRawTx(cmd.Context(), pk, tx)
+			if err != nil {
+				return err
+			}
+			// DISCUSS(#310): define UX for return values - should we return the raw response or a parsed/human readable response? For now, I am simply printing to stdout
+			fmt.Println(resp)
+
+			return nil
+		},
+	}
+	return unstakeCmd
+}
+
+func newUnpauseCmd(cmdDef actorCmdDef) *cobra.Command {
+	unpauseCmd := &cobra.Command{
+		Use:   "Unpause <fromAddr>",
+		Short: "Unpause <fromAddr>",
+		Long:  fmt.Sprintf(`Unpauses the %s actor with address <fromAddr>`, cmdDef.Name),
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// TODO(#150): update when we have keybase
```

Comment:
Is this okay that `exact args` is needed but we are reading from the PK file? @Olshansk WYT?

Reply:
I think what we should do is leave it as is but make sure the interim progress is clear & documented incase we get hit by the skateboard / bicycle / bike / car.

1. Since `args` isn't used, make it an anonymous variable via `_ []string`
2.Replace `cobra.ExactArgs(1)` with `cobra.ExactArgs(1) // REFACTOR(#150): Not being used at the moment. Update once a keybase is implemented.`

Applicable elsewhere as well.

Reply:
Related: https://github.com/pokt-network/pocket/pull/177#discussion_r1003555857

Reply:
Addressed in https://github.com/pokt-network/pocket/pull/177/commits/f75a872c9c6b033e956ad335c526eaace41aa670